### PR TITLE
Add functions for retrieving IP address from interface by family

### PR DESCRIPTION
**Pull request by pdf**
Adds `IPv4Address` and `IPv6Address` functions to containers.

### CODE REVIEW

**Comment by caglar10ur** [comment]
Thanks!

File: lxc_test.go [hunk 1]
```diff
@@ -1051,6 +1051,36 @@ func TestIPAddress(t *testing.T) {
...
 }
 
+func TestIPv4Address(t *testing.T) {
+	c, err := NewContainer(ContainerName)
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	if unprivileged() {
+		time.Sleep(3 * time.Second)
```
Comment (caglar10ur) [inline lxc_test.go:1061]: I merged it anyway but I really don't like sleep calls with random intervals like that. Is there a reason not to use WaitIPAddresses here?

File: lxc_test.go [hunk 2]
```diff
@@ -1051,6 +1051,36 @@ func TestIPAddress(t *testing.T) {
...
+}
+
+func TestIPv46ddress(t *testing.T) {
+	c, err := NewContainer(ContainerName)
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	if unprivileged() {
+		time.Sleep(3 * time.Second)
```
Comment (caglar10ur) [inline lxc_test.go:1076]: ^ ditto

File: lxc_test.go (hunk 1 above)
Comment (pdf) [inline lxc_test.go:1061]: Yeah, that wouldn't have been my choice either - I adapted these tests from the existing tests for related functionality already in the codebase :wink:

Comment (caglar10ur) [inline lxc_test.go:1061]: Doh, how did I forget that :) I'll take care of them soon...

**Pull request merged**